nvmf-autoconnect/NetworkManager/99-nvme-nbft-no-ignore-carrier.conf.in

```diff
@@ -0,0 +1,15 @@
+# Boot from NVMe over TCP (NBFT)
+#
+# For NVMe/TCP connections that provide namespaces containing rootfs
+# it is crucial to react on carrier events and reconnect any missing
+# NVMe/TCP connections as defined in the ACPI NBFT table. A custom
+# /usr/lib/NetworkManager/dispatcher.d/99-nvme-nbft-connect.sh hook
+# will respawn nvmf-connect-nbft.service on such occasion.
+
+[device-nbft-no-ignore-carrier]
```

Comment:
I'm confused about the name of this config section. It kind of implies the `ignore-carrier=no`, or what am I missing here?

I am not a NetworkManager expert, but intuitively I'd sort this into the "connection" section of NetworkManager.conf. Or, perhaps better, I'd invent a new per-connection property for it (this way you wouldn't assume a fixed naming scheme for the interfaces).